examples/gan/prepare_gan_images.py

```diff
@@ -1,54 +1,62 @@
 import tensorflow as tf
 import os
 import json
+import argparse
 from shutil import copyfile
 
+parser = argparse.ArgumentParser(
+    description="Process images and create metadata JSON file.")
+
+parser.add_argument("input_dir", type=str, default=".",
```

Comment:
Can these also be Google Buckets? Then maybe it should default to a public google bucket hosted by us

Reply:
Just tested it locally, GCS buckets work. But in our case, the label file has been generated for the images in our bucket and added. So there is not a lot of value unless you get the whole data yourself e.g. from the official repo

Reply:
Default bucket added in db78609